src/tree/AzureAccountTreeItemWithProjects.ts

```diff
@@ -127,10 +127,10 @@ export class AzureAccountTreeItemWithProjects extends AzureAccountTreeItemBase {
 async function getCompiledProjectPath(projectPath: string): Promise<string | undefined> {
     const projectLanguage: string | undefined = getWorkspaceSetting(projectLanguageSetting, projectPath);
     if (projectLanguage === ProjectLanguage.CSharp || projectLanguage === ProjectLanguage.FSharp) {
-        const projFileName: string | undefined = projectLanguage === ProjectLanguage.CSharp ? await tryGetCsprojFile(projectPath) : await tryGetFsprojFile(projectPath);
-        if (projFileName) {
-            const targetFramework: string = await getTargetFramework(path.join(projectPath, projFileName));
-            return path.join(projectPath, getDotnetDebugSubpath(targetFramework));
+        const projFiles: string[] = await dotnetUtils.getProjFiles(projectLanguage, projectPath);
+        if (projFiles.length === 1) {
+            const targetFramework: string = await dotnetUtils.getTargetFramework(path.join(projectPath, projFiles[0]));
+            return path.join(projectPath, dotnetUtils.getDotnetDebugSubpath(targetFramework));
         } else {
             throw new Error(localize('unableToFindProj', 'Unable to detect project file.'));
```

Comment:
Wouldn't you want to surppress this too?

Reply:
Interesting...looked into it and this error can only happen in the tree so there's no report issue button anyways

![Screen Shot 2020-03-10 at 2 02 16 PM](https://user-images.githubusercontent.com/11282622/76359074-c4f36e80-62d7-11ea-98c5-250a7f6913e3.png)